### DISCUSSION TITLE
Computing matrix elements with external polarized photons

### DISCUSSION THREAD

**Discussion by kbya99** [FeynCalc/feyncalc#216] [Q&A]
Hello FeynCalc experts,

I want to compute the matrix element squared of the (gamma gamma -> f fbar) process, with explicit polarizations set for the external particles. I've been able to compute the unpolarized M^2 without problems, and I also know that I can set fermion polarizations by simply adding projection operators before/after the spinors. How can I explicitly set the polarizations of the external photons as well? 

Any help is appreciated!

Thanks!

**Comment by vsht** [comment]
Hi,

> How can I explicitly set the polarizations of the external photons as well?

There is nothing built-in for that, so you would need to plug in explicit expressions by hand.
Discussion #156 might be also relevant.
